scripts/ci/setup_and_run_python.sh

```diff
@@ -1,2 +1,6 @@
 . ./setup.sh
+if [ -e "/home/${USER:0:1}/${USER}/.keytab" ]; then
+    echo "Initialize kerberos"
+    kinit -kt /home/${USER:0:1}/${USER}/.keytab ${USER}@CERN.CH
```

Comment:
Why is this needed here?

Reply:
This is needed to run the script "scripts/tests/run_GoF.sh" but maybe there is another way to do it?

Reply:
But in the ci the kerberos ticket is already initialized by a separate step no?

Reply:
I want to use the script/ci/setup_and_run_python.sh outside of the singularity in a bash script to run commands in wremnants singularity and cmssw-cc7, but for the wremnants singularity I was not able to get the kerberos initialized in any other way than this one. I tried to follow what is done in the CI, i.e. executing the following commands
```
scripts/ci/run_with_singularity.sh kinit -kt ~/.keytab dwalter@CERN.CH
scripts/ci/run_with_singularity.sh klist -k -t -e ~/.keytab
scripts/ci/run_with_singularity.sh xrdfs root://eoscms.cern.ch// ls /store/cmst3/group/wmass/w-mass-13TeV/NanoAOD
``` 
but I get a "Permission denied" error.